### ISSUE TITLE
One file is missing in the nuscenes_3d val directory.

### ISSUE THREAD

**Issue by RONINGOD** [pengsongyou/openscene#77]
not exist: cce9a0212eb1444696e2decbc95a015e

**Comment by pengsongyou** [comment]
Thanks for pointing this out. I just checked, it only exists in `nuscenes_2d`, not in `nuscenes_3d` and the extracted features. I guess that should not throw errors during evaluation?

**Comment by RONINGOD** [comment]
> Thanks for pointing this out. I just checked, it only exists in `nuscenes_2d`, not in `nuscenes_3d` and the extracted features. I guess that should not throw errors during evaluation?

Yes, I just want to verify that the openseg features generated by my code using the nuScenes dataset downloaded from the official website correspond to those provided by you.

**Comment by ayesha-ishaq** [comment]
Hi, @pengsongyou, you haven't released code to process the files directly from the original dataset, so could you publish the entire processed val set, including the missing scene? It would be a great help.
